test/nectar_test.exs

```diff
@@ -2,7 +2,11 @@ defmodule NectarTest do
   use ExUnit.Case
   doctest Nectar
 
+  test "a known good HTTP server" do
+    assert {:ok, %HTTPoison.Response{}} = HTTPoison.get("http://elixir-lang.org/")
```

Comment:
This is unlikely to break, but having a test like this is generally discouraged. It does not test anything in Nectar. The only things it tests are HTTPoison and the responsiveness of elixir-lang.org. I'd recommend removing this altogether.